### PR TITLE
Chore: Convert Wrapper to function component

### PR DESCRIPTION
**Pull request by Elfo404**
Convert Explore's `Wrapper.tsx` to a function component and refactor some logic. There are some easily achievable optimizations left, will create a follow-up PR, but opening this for review now to unblock @ifrost

### CODE REVIEW

File: public/app/features/explore/Wrapper.tsx [hunk 1]
```diff
@@ -36,64 +31,31 @@ const styles = {
...
     //We should probably abstract this out at some point
-    this.context.chrome.update({ sectionNav: this.props.navModel.node });
-    this.context.keybindings.setupTimeRangeBindings(false);
+    chrome.update({ sectionNav: navModel.node });
+  }, [chrome, navModel]);
 
+  useEffect(() => {
+    // FIXME: after cloasing the split view, changing time via keyboard causes an error (happens also on main)
+    // will create a separate issue for this.
+    keybindings.setupTimeRangeBindings(false);
```
Comment (Elfo404) [inline public/app/features/explore/Wrapper.tsx:55]: creating an issue for this :D

Comment (Elfo404) [reply public/app/features/explore/Wrapper.tsx:55]: https://github.com/grafana/grafana/issues/56175

**Review by gelicia — APPROVED** [review]
Played around with different split states, opening/closing splits, etc and all seemed to work fine. Great work!!

**Review by ifrost — APPROVED** [review]
LGTM 🔥

Tested manually:
- [x] New navigation / old navigation
- [x] Mixed / Non-mixed data sources
- [x] Split view
- [x] Creating panes from the URL
- [x] Updating document title

**Pull request merged**